registry.go

```diff
@@ -15,8 +15,7 @@ type FunctionMap = template.FuncMap
 // It also allows for easy extension of the template functions by adding a new one.
 type Registry interface {
 	// Uid returns the unique name of the registry. This name is used to identify
-	// the registry and in future prevent duplicate registry registration.
-	// TODO: Consider implement a solution for duplicate registry registration.
+	// the registry author and name and prevent duplicate registry registration.
 	Uid() string
```

Comment:
Not related to this PR

But here I would have preferred UID()

I tend to use revive var-naming rule for it

https://revive.run/r#var-naming

It can be easily added to golangci-lint

Reply:
Yes, I see that when change uid everywhere, I self-put the link to the style guide https://google.github.io/styleguide/go/decisions.html#initialisms 

I think this is the best moment to fix that before v1.0.0 after that that will be impossible to change without a breaking change

Reply:
Exactly